cargo-geiger/src/args.rs

```diff
@@ -164,6 +164,18 @@ impl Args {
                 None
             },
         };
+
+        if args.readme_args.update_readme
+            && args.charset != Charset::GitHubMarkdown
+        {
+            eprintln!(
+                "Charset has been specified as {:?}, but the `--update-readme` flag has also been provided. \
+                To ensure the report written to the README.md is correct, a reduced charset will be used.",
+                args.charset
+            );
```

Comment:
Maybe we should rename the `--charset` argument to `--stdout-charset` or `--terminal-charset` and ignore it completely for the `--update-readme` mode, what do you think?

Reply:
This sounds like a good idea to me! If we are going to add the HTML output as well, it would make more sense to have it all together.
I'll make that change now!